prow/cmd/splice/main.go

```diff
@@ -322,23 +324,26 @@ func main() {
 		if len(queue) == 0 {
 			continue
 		}
+		log.Infof("PRs in queue: %v", queue)
```

Comment:
I would prefer having this before the `len(queue) == 0` check, so you can see that splice is doing things each minute.